index.html

```diff
@@ -422,6 +422,14 @@ <h3>Proofs</h3>
         </p>
 
         <dl style="margin-left: 1em;">
+          <dt>id</dt>
+          <dd>
+An optional identifier for the proof, which MUST be a URL [[URL]], such as
+a UUID (`urn:uuid:6a1676b8-b51f-11ed-937b-d76685a20ff5`).
```

Comment:
```suggestion
a UUID as a URN (`urn:uuid:6a1676b8-b51f-11ed-937b-d76685a20ff5`).
```

Could reference this too: https://www.rfc-editor.org/rfc/rfc4122